cmd/configure/attributes/container/container.go

```diff
@@ -1,3 +1,4 @@
+// Package container provides utilities for handling container attributes.
```

Comment:
does not provide any interesting info

I am very reluctant to comment packages in general, looked around a bit in other packages, they not have it always. (`fmt` does not have one for example)

I good example where it make sense is the `context` [package](https://pkg.go.dev/context#pkg-overview) (the whole "overview" part is the comment on the package)